cli/types/cypress.d.ts

```diff
@@ -1916,11 +1916,12 @@ declare namespace Cypress {
      * Enables running Cypress commands in a secondary domain
      * @see https://on.cypress.io/switchToDomain
      * @example
-     *    cy.switchToDomain('example.com', { key: 'value' } ({ key }) => {
+     *    cy.switchToDomain('example.com', [{ key: 'value' }, 'foo'] ([{ key }, foo]) => {
      *      expect(key).to.equal('value')
+     *      expect(foo).to.equal('foo')
      *    })
      */
-    switchToDomain<T>(domain: string, data: T, fn: (data: T) => void): Chainable<undefined>
+    switchToDomain(domain: string, data: [], fn: (data: []) => void): Chainable<undefined>
```

Comment:
```suggestion
    switchToDomain<T>(domain: string, data: T[], fn: (data: T[]) => void): Chainable<undefined>
```

I think we can keep the generic for these like this, right?

Reply:
Yeah, you're right, we can keep the generic.

Reply:
[Updated here](d1d6599ea65312b86ae2ffcd9018178a1bf2c732)